packages/vue-query/README.md

```diff
@@ -0,0 +1,81 @@
+[![Vue Query logo](./media/vue-query.png)](https://damianosipiuk.github.io/vue-query/)
+
+[![npm version](https://img.shields.io/npm/v/vue-query)](https://www.npmjs.com/package/vue-query)
+[![npm license](https://img.shields.io/npm/l/vue-query)](https://github.com/DamianOsipiuk/vue-query/blob/main/LICENSE)
+[![bundle size](https://img.shields.io/bundlephobia/minzip/vue-query)](https://bundlephobia.com/result?p=vue-query)
+[![npm](https://img.shields.io/npm/dm/vue-query)](https://www.npmjs.com/package/vue-query)
```

Comment:
does this need to be adapted? it still references `https://damianosipiuk.github.io/vue-query/` and `vue-query` on bundlephobia and `npm`